### DISCUSSION TITLE
fitcontent with many data points, with restrict chart width.

### DISCUSSION THREAD

**Discussion by neobie** [tradingview/lightweight-charts#981] [Q&A]
How to fitcontent() with many data points ? Lets say, more than 1000 data points, 10 years data.

I notice when the chart shrink to smaller width, the chart is automatically truncated. 
Have no idea what is the limit of maximum points per width px lightweight-chart can handle to display.

![Screenshot 2022-01-21 at 10 27 17 AM](https://user-images.githubusercontent.com/4213973/150455206-2641fb00-e1da-450d-910b-38f8e23850cf.png)
![Screenshot 2022-01-21 at 10 27 32 AM](https://user-images.githubusercontent.com/4213973/150455219-09dba913-f221-4e2c-ac43-6051f33fcae4.png)

**Comment by timocov** [comment]
See https://tradingview.github.io/lightweight-charts/docs/api/interfaces/TimeScaleOptions#minbarspacing
